neptune/alpha/internal/containers/disk_queue.py

```diff
@@ -35,24 +40,28 @@ def __init__(
             log_files_name: str,
             to_dict: Callable[[T], dict],
             from_dict: Callable[[dict], T],
+            version_getter: [Callable[[T], int]],
             max_file_size: int = 64 * 1024**2):
         self._dir_path = dir_path
         self._log_files_name = log_files_name
         self._to_dict = to_dict
         self._from_dict = from_dict
+        self._version_getter = version_getter
         self._max_file_size = max_file_size
         self._event_empty = Event()
         self._event_empty.set()
+        self._last_ack_file = SyncOffsetFile("last_ack_version")
```

Comment:
This will create the offset file at ./last_ack_version, won't it?

Reply:
Yup. It's a bug.